### DISCUSSION TITLE
Scrypted HKSV Changes

### DISCUSSION THREAD

**Discussion by cheme75** [mrlt8/docker-wyze-bridge#387] [General]
Looks like the method outlines in the wiki fr scrypted will be changing due to a change starting in ios 15.5 - it is now going to be required to run the homekit plugin as a standalone for each cam. I had just managed to get the darn scrypted bridge working and now this change somehow leaves my cams unable to be detected by homekit. So I'm back to using HB ffmpeg which is ok, since it seems to be stable. I only have 3 cams, but if you got a lot its going to be royal pain to setup each using scrypted now.

**Comment by mrlt8** [comment]
Thanks for the heads up! Updating my scrypted now. 

The standalone mode seems similar to unbridge mode in homebridge.

Looks like switching modes will also allow for notifications when the cameras go offline.

**Reply by cheme75** [reply]
Yes, after you explained unbridged mode, this change is the same. I spent 2 days getting things working only hit a wall with 2 issues - transpose for the one cam does not work in the transcoding plugin and then I could not get any standalone cam to be recognized in home app. koush said it was my firewall and said something about port 8554 but I explained im not using rtsp fw, using the wyze bridge for rtsp and that is using 8554... anyway I got fed up with thwe time and effort to tweak each cam in the UI which requires an inordinate amt of time to flip back and forth between cams, plugins, etc - so removed the containers. Now that I found docker-compose on the linux pc, I'll use that to start a fresh container and re-tryne more time, eventually. I see I can get notifications in home app, but all other features require recording and I have no desire to subscribe to icloud+ so using the other features in hksv are not a big deal for me and I have no need for automation with these cams. I guess there are plugins and whatnot to create local recordings and maybe do facial recognition - so I'm really doing this for future possibilities and potentially loading the rtsp firmwaare to get out of the wyze system. But for now I might be just fine with hoobs and ffmpeg which seems to be fairly stable so far.

**Reply by mrlt8** [reply]
Haven't really tried it myself, but there's another homebridge plugin that offers support for HomeKit secure video called [homebridge-camera-ui](https://github.com/seydx/homebridge-camera-ui).

Again, not sure how useful it would be over camera ffmpeg if you don't have iCloud storage.

**Reply by cheme75** [reply]
ok, I'll take a look. I recall something about camera-ui but that was before I got this bridge working.

**Reply by cheme75** [reply]
> Again, not sure how useful it would be over camera ffmpeg if you don't have iCloud storage.

at present, I don’t need to pay for iCloud stg, the 5gb free handles my backups, but obviously too limited to handle photo and video stg. Everything else I use my free OneDrive or google drive stg or my own NAS via remote access. Like Wyze subscriptions, hksv requiring iCloud+ is no different - forced usage with no option to use our personal NAS, although iCloud+ might be more feature rich and possibly more reliable, and definitely more useful for a broad range of uses. From what I see so far, the hclx hack option still doesn’t overcome the use of iCloud+, just can avoid reliance on Wyze servers.
